doc/source/getting_started/index.md

```diff
@@ -5,5 +5,13 @@
 ```{toctree}
 Introduction
 XarrayforArviZ
-InferenceDataCookbook
+CreatingInferenceData
+WorkingWithInferenceData
 ```
+
+     4D            2C       2C         4D
+A 1001101 años del 101100 y 101100 del 1001101.
+
+
+44 2C
+77 4D
```

Comment:
looks like something got copy pasted unintentionally

Reply:
haha, at least is not something sensitive like my bank account password, haha